feat: transpose stream info.

zhyd1997 opened the pull request:
Fixes #731 

- [x] keep previous requirement:  sort the list by the user's flow rate first, and the TVS second.

zhyd1997 committed:
feat: added basic stream table UI.

mikeghen commented:
This is looking good, one thing to add would be to sort the list by the user's flow rate first, and the TVS second

zhyd1997 committed:
feat: added inflowRate.
- renamed Market.tsx to Markets.tsx.
- unexport map constant from Content.tsx.
+ added map constant to flowConfig indexIDA.

zhyd1997 committed:
feat: added streamed.
+ refactor: lifted streams query up.
+ feat: added calculatedStreamed utils.

zhyd1997 committed:
fix: streamed.

zhyd1997 committed:
feat: added tvs and streams.
+ feat: aded flowKey and type into indexIDA.
+ feat: added received [1/2].

zhyd1997 committed:
feat: added input and output token balances.

zhyd1997 committed:
fix: marked streamed-in-token and distributed-out-token balances as optional to added placeholder for better UX.

zhyd1997 committed:
feat: added received [2/2].

zhyd1997 committed:
feat: new market supported launchpad.

+ exported IndexIDAType from `flowConfig` file.

zhyd1997 committed:
fix: added contract address link.

zhyd1997 committed:
feat: added skeleton when loading table data.

zhyd1997 committed:
fix: contract addresses.
+ fix: replaced <a> with mui Link component and removed unused markets.module.scss.

zhyd1997 committed:
feat: added price field.
+ chore: exported POOLS from 'flowConfig' file.
+ chore: added Pool type.
+ chore: added 'type' and 'pool' fields in 'Row' interface.
+ feat: added 'PricePlaceholder' component.
+ refactor: added getLauchpadPrice utils fn.
- fix: removed unused 'price' field in 'Row' interface.

zhyd1997 committed:
fix: set streamed and received value decimals.

zhyd1997 committed:
fix: disabled get coingekco price on redux-saga.
* we have used reservedUSD in liquidity pools.

zhyd1997 committed:
feat: added tooltips for operations.
+ refactor: added Operation component.

zhyd1997 committed:
feat: supported CURD stream.
+ feat: added StreamModal component.
+ feat: enhanced stream operation feedback (imported Toast).
+ chore: added more fields into Row interface.

zhyd1997 committed:
fix: enhanced error handler.

zhyd1997 committed:
chore: added Config type.

zhyd1997 committed:
refactor: exported Row interface as separte file.

zhyd1997 committed:
fix: moved old streams page into markets.
+ fix: enhanced tvs dispaly.

zhyd1997 committed:
feat: show dexscreener link.
refactor: added LinkPlaceholder component.

zhyd1997 committed:
fix: removed navlnk padding-top on tablet device.

zhyd1997 committed:
Merge branch 'v2' into feat/transpose-stream-info

zhyd1997 committed:
fix: sort list by tvs when wallet is not connected.

abhisheknaiidu reviewed (approved):
LGTM 🚀
